docs/konnectors.md

```diff
@@ -320,18 +320,40 @@ On the host create the file /etc/rkt/net.d/10-containers.conf
 
 and run your container with the "--net=bridge" option. That way, a new interface is available in the container and gives you access to the host.
 
+## Konnector install and run details
+
+### Install
+
+The konnectors will be installed in the .cozy_konnectors directory (not in VFS ?) using npm install
```

Comment:
Using npm packages is mainly to get the openness with the many possibilities of hosting (github, gitlab, bitbucket, gist, etc.... and it also offers the versioning. It seems that yarn could be the response to the points you raise. And nodejs and yarn which would be used to get and update the packages would not need to be updated really often, it would not be it, which would run konnectors. We could even imagine using the rkt container to fetch the packages (then no need install nodejs with cozy-stack).

Especially for the konnectors, we need to ease the development and hosting to keep the very active community of konnectors developers. A github repo is still (to my view) the easiest way of hosting a konnector and it is compatible with yarn. But I suppose using only github repos as the only source of packages is still OK and would also allow the versioning as for the applications.